src/main/webapp/js/diagramly/sidebar/Sidebar-My.js

```diff
@@ -17,6 +17,84 @@
     }, '');
   };
 
+  Sidebar.prototype.createOnCRUDAction = (sidebar, type) => {
+    function onCRUDActionTitle(type) {
+      makeTitle = (type) => {
+        if (type === "create") {
+          return "Create";
+        } else if (type === "read") {
+          return "Read";
+        } else if (type === "update") {
+          return "Update";
+        } else if (type === "delete") {
+          return "Delete";
+        }
+      }
+      return (`On ${makeTitle(type)} Action`);
```

Comment:
You can simplify this with a simple object:

```suggestion
      const title = {
        "create": "Create",
        "read": "Read",
        "update": "Update",
        "delete": "Delete",
      }[type];
      return (`On ${title} Action`);
```